### ISSUE TITLE
There is a significant difference in the log sampling effect between manual reasoning generation and the training process.

### ISSUE THREAD

**Issue by Qnh233** [CompVis/stable-diffusion#902]
Hello, in the sampling records of my log during the training process, I found that the random sampling for generating samples works very well. However, when I use txt2img or img2img for inference, the results are very poor. Even when I use samples from the training set for manual inference, the results are also very bad. It seems that it has collapsed, with a little semantic understanding but not fully grasped. It's like the brain knows what it should be but is twitching randomly, resulting in a messy generation.
I have tried many things, but I still can't solve this problem. Besides, did the default training use full precision? Sometimes when I use manual inference, I get abnormal values when using automatic mixed precision, but it works fine when using full precision.
Thank you all. This is very important to me. Thank you! Additionally, I used the loss function of the GAN, but I saw online that when using the dual optimizer of PL, automatic optimization needs to be stopped. But I saw that the source code of PL should support this. This is also one of my doubts.
